content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -339,6 +339,9 @@ In this subsection, perform the same steps as in the [Generate client and server
 
 ### Configure traffic for the `bookinfo.com` host
 
+> {{< warning_icon >}} port name must be distinct for HTTPS.
```

Comment:
This warning doesn't seem appropriate here. Which port name is it referring to? I don't see an instruction that requires the user to name a port.

Reply:
In the gateway definition yaml file. Don’t seen other place to add this comment. Let me know if you have any idea

Reply:
I think the place to document this issue is [here](https://istio.io/help/ops/traffic-management/deploy-guidelines/).

Reply:
Maybe also mention the restriction in the description of the `port` property of ref docs?
https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Server

Reply:
I can but for my understanding "Configure a TLS ingress gateway for multiple hosts" and add a warning on this section is the correct one. This issue only apply when you deploy multiple hosts with same port name for https on the gateway.

Reply:
@rshriram told me we may be able to remove this restriction from the code, so we won't need anything in the doc. I reopend the istio/istio issue.

Reply:
Ok, let me know if you want to close the 2 PR has requested